Demo/Demo.swift

```diff
@@ -73,6 +74,9 @@ public enum ComponentViews: String, CaseIterable {
             return secondViewController
         case .easterEggButton:
             return DemoViewController<EasterEggButtonDemoView>()
+        case .bottomSheet:
+            let controller = DemoViewController<ButtonDemoView>()
```

Comment:
Not sure if it's a problem or not, but does it play well with double tap gestures we're using to dismiss demo controller? Maybe it's better to have a view with "present" button like you did in FinnLabs project.

Reply:
It doesn't seem to be a problem at the moment. It will just dismiss if you double tap inside the bottom sheet. Can of course consider having a separate demo view controller for the bottom sheet